<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PulseDialog</class>
 <widget class="QDialog" name="PulseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NeXtRAD Dialog</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="toolTipDuration">
   <number>0</number>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>560</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pulse Parameters</string>
   </property>
  </widget>
  <widget class="QComboBox" name="waveform_comboBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QTextEdit" name="num_pris">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>120</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="samples_per_pri">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>160</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="dac_delay">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>200</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="adc_delay">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>240</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="pol_order">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>340</y>
     <width>281</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;pre class=&quot;tab&quot;&gt;&lt;span style=&quot; font-family:'monospace'; font-weight:600; color:#008000;&quot;&gt;Mode     FreqBand     TxPol     RxPol&lt;/span&gt;&lt;/pre&gt;&lt;/p&gt;&lt;p&gt;&lt;pre class=&quot;tab&quot;&gt;&lt;span style=&quot; font-family:'monospace'; font-weight:600; color:#008000;&quot;&gt;0          &lt;/span&gt;&lt;span style=&quot; font-family:'monospace'; color:#008000;&quot;&gt;L           V          V&lt;/span&gt;&lt;/pre&gt;&lt;/p&gt;&lt;p&gt;&lt;pre class=&quot;tab&quot;&gt;&lt;span style=&quot; font-family:'monospace'; font-weight:600; color:#008000;&quot;&gt;1          &lt;/span&gt;&lt;span style=&quot; font-family:'monospace'; color:#008000;&quot;&gt;L           V          H&lt;/span&gt;&lt;/pre&gt;&lt;/p&gt;&lt;p&gt;&lt;pre class=&quot;tab&quot;&gt;&lt;span style=&quot; font-family:'monospace'; font-weight:600; color:#008000;&quot;&gt;2          &lt;/span&gt;&lt;span style=&quot; font-family:'monospace'; color:#008000;&quot;&gt;L           H          V&lt;/span&gt;&lt;/pre&gt;&lt;/p&gt;&lt;p&gt;&lt;pre class=&quot;tab&quot;&gt;&lt;span style=&quot; font-family:'monospace'; font-weight:600; color:#008000;&quot;&gt;3          &lt;/span&gt;&lt;span style=&quot; font-family:'monospace'; color:#008000;&quot;&gt;L           H          H&lt;/span&gt;&lt;/pre&gt;&lt;/p&gt;&lt;p&gt;&lt;pre class=&quot;tab&quot;&gt;&lt;span style=&quot; font-family:'monospace'; font-weight:600; color:#008000;&quot;&gt;4          &lt;/span&gt;&lt;span style=&quot; font-family:'monospace'; color:#008000;&quot;&gt;X           VH         V&lt;/span&gt;&lt;/pre&gt;&lt;/p&gt;&lt;p&gt;&lt;pre class=&quot;tab&quot;&gt;&lt;span style=&quot; font-family:'monospace'; font-weight:600; color:#008000;&quot;&gt;5          &lt;/span&gt;&lt;span style=&quot; font-family:'monospace'; color:#008000;&quot;&gt;X           HH         V&lt;/span&gt;&lt;/pre&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="toolTipDuration">
    <number>0</number>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
   <property name="lineWrapColumnOrWidth">
    <number>0</number>
   </property>
  </widget>
  <widget class="QTextEdit" name="pre_pulse">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>430</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="pri">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>390</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="l_band_freq">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>470</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="x_band_freq">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>510</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>291</width>
     <height>491</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="waveform_label">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Waveform Index (1-14):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="num_pris_label">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Number of PRIs:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="samples_per_pri_label">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Samples per PRI:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="dac_delay_label">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>DAC Delay (No of 180 MSPS):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="adc_delay_label">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ADC Delay (No of 180 MSPS):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="pol_order_label_2">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Pol Order: Please select numbers:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="pol_order_label">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Pol Order (e.g. &quot;0,1,2,3,4,5&quot;):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="pri_label">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>PRI (usecs):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="pris_label_2">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Pre Pulse (usecs):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="pris_label_3">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>L Band Frequency (MHz):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="pris_label_4">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>X Band Frequency (MHz):</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>236</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioButton0">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="radioButton1">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QRadioButton" name="radioButton2">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QRadioButton" name="radioButton3">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QRadioButton" name="radioButton4">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QRadioButton" name="radioButton5">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PulseDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PulseDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
